src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Android/KnownPackages.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
+using System;
```

Comment:
I don't know what happened here. It's not whitespace at the end

Reply:
Changed BOM?

Reply:
probably but:
```
@@@ -1,4 -1,4 +1,4 @@@
--<U+FEFF><U+FEFF>using System;
++<U+FEFF>using System;
```
I don't know enough unicode for this but I thought BOMs are either 2 bytes or just not there. I'll try to figure it out

Reply:
We somehow lost the `KnownPackages.Acr_UserDialogs` NuGet in this file: https://github.com/xamarin/xamarin-android/commit/1d608bf194f5cddcb4d61a4bca3602f920da5246#diff-4469ed45319624df448a82ed4bc98b6d

Reply:
Yeah that was my fault, I botched a grep and thought it wasn't actually there on master. I am going to rebase away that merge and do again

Reply:
@jonathanpeppers actually I didn't botch the grep, I'm confused: https://github.com/xamarin/xamarin-android/blob/d700d7a16642d90b0dd08668c12fbad25769c391/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Android/KnownPackages.cs

It doesn't seem to be on master?

Reply:
Oh! It's a commit you added on `mono-2018-10`, not on master. That's why I didn't find it.

Reply:
To me it looks like the BOM was bad and is now fixed by how my mergetool (`opendiff`) saves, but I am happen to be taught otherwise. The issue survived a re-do of the merge. If it doesn't affect the build let's please not let it block the merge, and fixup on master :)

Reply:
@jonpryor The "maybe bad" BOM was introduced here, also as a strange one-line change: https://github.com/xamarin/xamarin-android/commit/1b5949c198e7ee4a39de1b645ffc597b7fe5697e